Corona-Warn-App/src/main/java/de/rki/coronawarnapp/statistics/local/source/LocalStatisticsProvider.kt

```diff
@@ -110,6 +110,7 @@ class LocalStatisticsProvider @Inject constructor(
     fun clear() {
         Timber.d("clear()")
 
+        localStatisticsConfigStorage.activeDistricts.update { emptySet() }
```

Comment:
Id rather follow the pattern of having a clear method on `localStatisticsConfigStorage` that deletes the whole shared prefs like in `CWASettings` and our other storage classes, the provider does not need to know how to specifically reset the settings

Reply:
updated